api/v1alpha1/system_types.go

```diff
@@ -833,6 +876,11 @@ func (tc *ThinkingSpec) Default() {
 
 // SystemRailsConsoleSpec configures the App component of System
 type SystemRailsConsoleSpec struct {
+	// Image specification for the Sphinx component.
```

Comment:
```suggestion
	// Image specification for the Console component.
```

Reply:
Fixed in https://github.com/3scale-ops/saas-operator/pull/257/commits/88e06aa1ef222ee65548ee97056e514b6a22660b